media-gfx/qview/qview-3.0.recipe

```diff
@@ -1,20 +1,28 @@
 SUMMARY="Minimalist image viewer"
-DESCRIPTION="qView is an image viewer designed with minimalism and usability in mind."
+DESCRIPTION="qView is an image viewer designed with minimalism and usability \
+in mind. \
+qView was designed from the very start to be as visually minimal and space \
+efficient as possible. No cluttered interface, just your image and a titlebar.\
+qView opens practically instantly, and switching images is just as quick, all \
+while having low memory and CPU usage. \
```

Comment:
I'd suggest to remove the " `\`" here and instead even insert an empty new line to get a real paragraph.